From f1d81469a61ed25b80d4a6e79635a2940abaf1e2 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 8 Nov 2013 10:28:48 +0100
Subject: [PATCH 277/509] dma: xilinx: Fix compilation warning for x86

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit 08b7c5cde9c5ac0fcfa42a990b2d53a08fe231c5

Zero day testing system reports compilation failures
for dma drivers because of missing header and incorrect
printk messages.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/dma/xilinx/vdmatest.c       |    6 +++---
 drivers/dma/xilinx/xilinx_axicdma.c |    1 +
 drivers/dma/xilinx/xilinx_axidma.c  |    1 +
 drivers/dma/xilinx/xilinx_axivdma.c |    1 +
 4 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/drivers/dma/xilinx/vdmatest.c b/drivers/dma/xilinx/vdmatest.c
index f6225cd..f4debd1 100644
--- a/drivers/dma/xilinx/vdmatest.c
+++ b/drivers/dma/xilinx/vdmatest.c
@@ -275,7 +275,7 @@ static int vdmatest_slave_func(void *data)
 			dma_srcs[i] = dma_map_single(tx_dev->dev, buf, len,
 							DMA_MEM_TO_DEV);
 			pr_debug("src buf %x dma %x\n", (unsigned int)buf,
-							dma_srcs[i]);
+				 (unsigned int)dma_srcs[i]);
 			sg_dma_address(&tx_sg[i]) = dma_srcs[i];
 			sg_dma_len(&tx_sg[i]) = len;
 		}
@@ -286,8 +286,8 @@ static int vdmatest_slave_func(void *data)
 							test_buf_size,
 							DMA_DEV_TO_MEM);
 			pr_debug("dst %x dma %x\n",
-					(unsigned int)thread->dsts[i],
-					dma_dsts[i]);
+				 (unsigned int)thread->dsts[i],
+				 (unsigned int)dma_dsts[i]);
 			sg_dma_address(&rx_sg[i]) = dma_dsts[i];
 			sg_dma_len(&rx_sg[i]) = len;
 		}
diff --git a/drivers/dma/xilinx/xilinx_axicdma.c b/drivers/dma/xilinx/xilinx_axicdma.c
index c4417a4..7ebb68b 100644
--- a/drivers/dma/xilinx/xilinx_axicdma.c
+++ b/drivers/dma/xilinx/xilinx_axicdma.c
@@ -19,6 +19,7 @@
 #include <linux/init.h>
 #include <linux/interrupt.h>
 #include <linux/io.h>
+#include <linux/irqdomain.h>
 #include <linux/module.h>
 #include <linux/of.h>
 #include <linux/of_address.h>
diff --git a/drivers/dma/xilinx/xilinx_axidma.c b/drivers/dma/xilinx/xilinx_axidma.c
index 0a922ec..c524c9f 100644
--- a/drivers/dma/xilinx/xilinx_axidma.c
+++ b/drivers/dma/xilinx/xilinx_axidma.c
@@ -22,6 +22,7 @@
 #include <linux/init.h>
 #include <linux/interrupt.h>
 #include <linux/io.h>
+#include <linux/irqdomain.h>
 #include <linux/module.h>
 #include <linux/of.h>
 #include <linux/of_address.h>
diff --git a/drivers/dma/xilinx/xilinx_axivdma.c b/drivers/dma/xilinx/xilinx_axivdma.c
index 400a5b4..131a93e 100644
--- a/drivers/dma/xilinx/xilinx_axivdma.c
+++ b/drivers/dma/xilinx/xilinx_axivdma.c
@@ -24,6 +24,7 @@
 #include <linux/interrupt.h>
 #include <linux/dmapool.h>
 #include <linux/io.h>
+#include <linux/irqdomain.h>
 #include <linux/of.h>
 #include <linux/of_platform.h>
 #include <linux/platform_device.h>
-- 
1.7.5.4

